components/sections/call_to_action/variant_e.tsx

```diff
@@ -1,255 +1,199 @@
-                    }
-                    className="mb-4 w-full rounded bg-webriq-blue py-4 text-sm font-bold leading-normal text-white transition duration-200 hover:bg-webriq-darkblue"
-                    type="submit"
-                  >
-                    {form?.buttonLabel}
-                  </button>
+    <section className="py-20 bg-gray-50">
+      <Container maxWidth={448}>
+        {form?.fields && <SignUpForm form={form} signInLink={signInLink} />}
+        {formLinks && (
+          <Flex wrap align="center" justify="center" c>
+            {formLinks?.map((link, index, { length }) => (
+              <div key={index}>
+                <Link
+                  href={extractLink(link)}
+                  className="text-sm font-bold text-primary hover:text-primary-foreground"
+                >
+                  {link?.label}
+                </Link>
+                {index === length - 1 ? null : index === length - 2 ? (
+                  <span>&nbsp;and&nbsp;</span>
+                ) : (
+                  <span>&nbsp;,&nbsp;</span>
                 )}
-              </WebriQForm>
-              {signInLink?.label && (
-                <p className="text-xs text-gray-500">
-                  <span>Already have an account?</span>
-                  <ConditionalLink
-                    link={signInLink}
-                    className="text-webriq-darkblue hover:text-webriq-babyblue"
-                    ariaLabel={signInLink?.label}
-                  >
-                    {signInLink?.label}
-                  </ConditionalLink>
-                </p>
-              )}
-            </div>
-          )}
-          {formLinks && (
-            <div className="flex flex-wrap items-center justify-center text-sm text-gray-500">
-              {formLinks?.map((link, index, { length }) => (
-                <div key={index}>
-                  <ConditionalLink
-                    link={link}
-                    className="text-webriq-darkblue hover:text-webriq-blue font-bold"
-                    ariaLabel={link?.label}
-                  >
-                    {link?.label}
-                  </ConditionalLink>
-                  {index === length - 1 ? null : index === length - 2 ? (
-                    <span>&nbsp;and&nbsp;</span>
-                  ) : (
-                    <span>&nbsp;,&nbsp;</span>
-                  )}
-                </div>
-              ))}
-            </div>
-          )}
-        </div>
-      </div>
+              </div>
+            ))}
+          </Flex>
+        )}
+      </Container>
     </section>
   );
 }
 
-/**
- *
- * @param {fields}
- * @returns input fields according to type
- */
-function FormFields({ fields }: { fields: FormFields }) {
-  const [value, setValue] = React.useState(null); // setting selected value for input field radio type
-  const [checked, setChecked] = React.useState([]); // setting selected value for input field checkbox type
+// /**
```

Comment:
Could we remove this block of code if it will not be used? I think this has been replaced by the Form UIs defined in Storybook.